network/handlers/drain_test.go

```diff
@@ -236,6 +236,58 @@ func TestDrainMechanics(t *testing.T) {
 	}
 }
 
+func TestDrainerKProbe(t *testing.T) {
+	var (
+		w          http.ResponseWriter
+		req        = &http.Request{}
+		kprobehash = "hash"
+		kprobe     = &http.Request{
+			Header: http.Header{
+				"K-Network-Probe": []string{"probe"},
```

Comment:
Could you use the const value `K-Network-Probe`, `K-Network-Hash` and `probe` defined in network/network.go?

Reply:
ok, thanks for review😋